puppet-client.html

```diff
@@ -29,6 +29,49 @@
 
 @demo demo/index.html
 -->
+<dom-module id="puppet-client">
+    <template>
+        <style>
+            .box-container {
+                position: fixed;
+                top: 0;
+                left: 0;
+                width: 100%;
+                display: flex;
+                justify-content: center;
+            }
+            .box-container[hidden] {
+                display: none;
+            }
+            .box {
+                width: 50%;
+                border-radius: 0 0 10px 10px;
+                border: thin solid;
+                border-top: none;
+                text-align: center;
+                padding: 5px 0;
+            }
+            @media only screen and (max-width: 640px) {
+                .box { width: 100%; }
+            }
+            .notice {
+                background-color: #fff4cc;
+                border-color: #ffe070;
+            }
+            .error {
+                background-color: #fad7d4;
+                border-color: #f08a81;
+            }
+            .error pre {
+                white-space: pre-wrap;
+                text-align: left;
+                padding: 0 15px;
+            }
+        </style>
+        <div class="box-container" hidden$="{{!showReconnectingIn}}"><div class="box notice"><strong>Not connected.</strong> Connecting in {{reconnectionSeconds}}s... <a href="" on-tap="reconnectNow">Retry now</a></div></div>
```

Comment:
I think if we provide means to override this behavior (what you proposed with events) that would be enough